### PR TITLE
6: Contact Page Visuals

### LINKED ISSUE #6: Contact Page Visuals
This ticket doesn't concern with the functionality of the email form at the bottom. Just ensure that it looks like the demo website.
 
- For the fields, just use inputs and style them properly. You don't need to capture these inputs or have a form for them.

Image
(Ignore the Grammarly icon in the text area for `Message`, whoops...)

### PR DESCRIPTION
**Pull request by aumsoni2002**
image

Compare the styling with the old page.
This is issue: #6

### CODE REVIEW

**Comment by aumsoni2002** [comment]
I have added the font 'Share Tech' as a className to be used anywhere in side the project: 
globals.css
image

tailwind.config.ts
image

How to use:
`<div className="font-shareTech">...</div>`

**Comment by CrisH2307** [comment]
I recognize that you used config as a solution which is great, but you have to clarify the classname every div that you created.
I would suggest to you for my solution on #11 . Which you dont need to call the classname `font-shareTech` everytime

**Review by CrisH2307 — COMMENTED** [review]
You don't need to change the `global.css` and `layout.tsx` since it fixed at #11 

You can update the code one more time, the overall of the contact page is good to merge

File: app/contact/page.tsx [hunk 1]
```diff
@@ -1,3 +1,135 @@
...
-  return null;
+import React from "react";
+import Image from "next/image";
+import discordImg from "../../public/img/discord.png";
+import instagramImg from "../../public/img/insta-color.png";
+import linkedinImg from "../../public/img/linkedin.png";
+
+function Contact() {
+  return (
+    <div className="bg-[#1e1e1e] text-white font-shareTech min-h-screen flex flex-col justify-center items-center pt-[8rem]">
```
Comment (KaosElegent) [inline app/contact/page.tsx]:
```suggestion
    <div className="bg-[#1e1e1e] text-white min-h-screen flex flex-col justify-center items-center pt-[8rem]">
```
Due to the work done in #11, removing the explicit font set using tailwind might be better.

File: app/globals.css
Comment (KaosElegent) [inline app/globals.css:1]: The changes in this file and `tailwind.config.ts` aren't necessary anymore due to the work done for #8 in #11

**Review by KaosElegent — CHANGES_REQUESTED** [review]
Looks good, but seeing as there's a ticket specifically for the font update, might be better to handle it there.

**Review by KaosElegent — APPROVED** [review]

**Pull request merged**